Userland/Libraries/LibWeb/LocalStorageObject.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022, Valtteri Koskivuori <vkoskiv@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <LibIPC/Forward.h>
+
+namespace Web::LocalStorage {
```

Comment:
In LibWeb, namespaces are typically organized by specification. There is no "local storage" specification, as web storage is part of the main HTML specification. For that reason, this should be in the `Web::HTML` namespace (and also in the `LibWeb/HTML/` directory).

Also, this doesn't really seem to map to an existing concept in the spec. Would it make sense to pass it across the IPC boundary as a simple `OrderedHashMap<String, String>` instead? That's what `Web::HTML::Storage` uses for internal storage currently anyway.